### PR TITLE
Update sleep duration units and help command messages

### LINKED ISSUE #298: Sleep duration units given in Plural for value 1
image

1 Days, 1 Hours --> 1 Day, 1 Hour
1 Minutes --> 1 Minute

### LINKED ISSUE #303: Update help command messages for sleep

### PR DESCRIPTION
**Pull request by DaDevChia**
This pull request updates the sleep duration units to be given in singular form when the value is 1, and in plural form for all other values. It also updates the help command messages for sleep to include new commands for setting, editing, and listing sleep goals.

Closes #303
Closes #298

### CODE REVIEW

File: src/main/java/athleticli/data/sleep/Sleep.java [hunk 1]
```diff
@@ -107,15 +107,34 @@ public String generateSleepingDurationStringOutput() {
         Duration tempDuration = sleepingDuration;
         String sleepingDurationOutput = "";
         if (tempDuration.toDays() != 0) {
```
Comment (yicheng-toh) [inline src/main/java/athleticli/data/sleep/Sleep.java:109]: Perhaps using guard clause would be better?

File: src/main/java/athleticli/data/sleep/Sleep.java [hunk 2]
```diff
@@ -107,15 +107,34 @@ public String generateSleepingDurationStringOutput() {
...
+
+            if (tempDuration.toDays() == 1) {
+                sleepingDurationOutput += tempDuration.toDays() + " Day ";
+            } else {
+                sleepingDurationOutput += tempDuration.toDays() + " Days ";
+            }
+
             tempDuration = tempDuration.minusDays(tempDuration.toDays());
         }
         if (tempDuration.toHours() != 0) {
```
Comment (yicheng-toh) [inline src/main/java/athleticli/data/sleep/Sleep.java:119]: Could this be in guard clause too?

**Review by yicheng-toh — COMMENTED** [review]
Code is neat. Keep it up!

**Review by yicheng-toh — APPROVED** [review]
LGTM!

**Review by skylee03 — APPROVED** [review]
LGTM!

**Pull request merged**